sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -6,10 +6,20 @@
 
 - Supported `withCopyStart` method in `Snapshot` for copying incremental snapshot from incremental snapshot.
 - Supported `awaitCopyStartCompletion` and `awaitCopyStartCompletionAsync` method in `Snapshot`.
-- Supported `copyCompletionPercent` and `copyCompletionError` method in `Snapshot` for retrieving `CopyStart` progress. 
+- Supported `copyCompletionPercent` and `copyCompletionError` method in `Snapshot` for retrieving `CopyStart` progress.
 - Supported `withTrustedLaunch`, `withSecureBoot` and `withVTpm` methods in `VirtualMachine`.
 - Supported `UBUNTU_SERVER_18_04_LTS_GEN2`, `UBUNTU_SERVER_20_04_LTS` and `UBUNTU_SERVER_20_04_LTS_GEN2` in `KnownLinuxVirtualMachineImage`.
 
+### Breaking Changes
+
+- Property `uri` removed from `GalleryArtifactVersionSource`. This property was non-functional.
```

Comment:
nit, this may affect customer calling `serviceClient` here:
https://github.com/Azure/azure-sdk-for-java/issues/31860#issuecomment-1301728899

though I doubt their code ever run. I will provide them with an example with convenience code.

Reply:
Got it. Thought about whether to rename the new GalleryDiskImageSource to previous GalleryArtifactVersionSource (as it got uri but one more property).
And it might get less breaks.

However, this could make the model definition in Java too diverged from what intended by service.